site/src/css/components/_Carousel.scss

```diff
@@ -4,23 +4,19 @@
 
 .carousel-container {
     width: 100%;
-    height: 550px;
+    height: 36.2rem;
     display: flex;
     flex-direction: column;
     margin: 2em 0;
 
     @include mobile-view {
-        height: 300px;
+        height: 19.8rem;
```

Comment:
Thinking out loud here... can the style properties that are proportional (height, font-size) be declared as:
```
$base-font-size: 2.2em;
.base-container{
    font-size: $base-font-size;
    .something-smaller {
         font-size: $base-font-size - 0.2em; or $base-font-size - math.div($base-font-size , 0.11em)
    }
}
```
My syntax may not be accurate but hope it makes sense-:)